redvanworkshop/website
# Mk 140

#### What's this PR do?

This PR adds new cities to our employee map on the careers page. This also adds a python script that I wrote to make adding more cities to the map easier in the future. The script also comes with a readme file.

#### Where should the reviewer start?

For the actual changes on the website start at the Careers page.

#### How should this be manually tested?

Go to the careers page and see if the changes appear for the new cities. Below is an excel file of all of the cities that should show:

[RVWStates.xls](https://github.com/redvanworkshop/website/files/6543184/RVWStates.xls)

It may be easier to find these changes in the src/js/site.js file as this has a commented list of every coordinate in the markers variable.

For the script this can be found in a new directory I made in the src directory named "tools". Inside are some example images and a README.md. Follow the README file to see if you are able to get the script to work on your machine with the above excel document.

#### Any background context you want to provide?

This is the first script/readme that I have made. If there are any changes that you see that should be made please let me know.

#### What are the relevant jira issue?

MK-140

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/79174401/119587726-46c89e00-bd9d-11eb-96db-a1cbf35410d4.png)

![image](https://user-images.githubusercontent.com/79174401/119587780-619b1280-bd9d-11eb-986f-fca945e3bbc5.png)
